CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ This file follows the best practices from [keepachangelog.com](http://keepachang
 - Changed ActiveRecord schema format to :sql [#2531](https://github.com/sharetribe/sharetribe/pull/2531)
 - Upgraded Paperclip and Delayed::Paperclip, dropped deprecated AWS SDK v1 [#2522](https://github.com/sharetribe/sharetribe/pull/2522)
 - Upgraded mysql2 dependency [#2565](https://github.com/sharetribe/sharetribe/pull/2565)
+- Deprecated use of MySQL server version 5.6.x [2566](https://github.com/sharetribe/sharetribe/pull/2566)
```

Comment:
Create a new heading Deprecated for this one: http://keepachangelog.com/en/0.3.0/